docs/architecture.md

```diff
@@ -1,103 +1,172 @@
 
-Converse methods must return a Boolean value. True if an utterance was handled, otherwise False.
+OVOS provides a set of public TTS servers hosted by community members.  It uses the [ovos-tts-server-plugin](https://github.com/OpenVoiceOS/ovos-tts-server-plugin), and no additional configuration is needed.
 
-### Active Skill List
+[Supported TTS Plugins](https://github.com/orgs/OpenVoiceOS/repositories?q=ovos-tts-plugin&type=all&language=&sort=)
 
-A Skill is considered active if it has been called in the last 5 minutes.
+**TODO** Fix link to how to change tts
 
-Skills are called in order of when they were last active. For example, if a user spoke the following commands:
+[Changing TTS Plugin](change-tts.md)
 
-> Hey Mycroft, set a timer for 10 minutes
->
-> Hey Mycroft, what's the weather
+## PHAL
 
-Then the utterance "what's the weather" would first be sent to the Timer Skill's `converse()` method, then to the intent service for normal handling where the Weather Skill would be called.
+https://github.com/OpenVoiceOS/ovos-PHAL
 
-As the Weather Skill was called it has now been added to the front of the Active Skills List. Hence, the next utterance received will be directed to:
+PHAL stands for Plugin based Hardware Abstraction Layer and is used to allow access of different hardware devices to use the OVOS software stack.  It completely replaces the
+concept of hardcoded "enclosure" from mycroft-core.
 
-1. `WeatherSkill.converse()`
-2. `TimerSkill.converse()`
-3. Normal intent parsing service
+Any number of plugins providing functionality can be loaded and validated at runtime, plugins can
+be [system integrations](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-system) to handle things like reboot and
+shutdown, or hardware drivers such as [mycroft mark1 plugin](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-mk1)
 
-### Making a Skill Active
+[Supported PHAL Plugins](https://github.com/orgs/OpenVoiceOS/repositories?q=ovos-phal-plugin&type=all&language=&sort=)
 
-There are occasions where a Skill has not been triggered by the User, but it should still be considered "Active".
+**TODO** link to phal plugin details
 
-In the case of our Ice Cream Skill - we might have a function that will execute when the customers order is ready. At this point, we also want to be responsive to the customers thanks, so we call `self.make_active()` to manually add our Skill to the front of the Active Skills List.
+[PHAL Plugins](phal-plugins.md)
 
-## GUI Protocol
+### Admin PHAL
 
-T[he [gui service](https://github.com/OpenVoiceOS/ovos-core/tree/dev/mycroft/gui) in ovos-core will expose a websocket to
-the GUI clients following the protocol outlined [here](https://github.com/MycroftAI/mycroft-gui/blob/master/transportProtocol.md)
+Similar to regular PHAL, but is used when `sudo` or `privlidged user` is needed
```

Comment:
Suggest adding a warning NOT to use this unless it is absolutely necessary. Also consider mentioning the average user won't need to configure this.

Reply:
might check this one and see if that is enough